docs/source/tutorials.rst

```diff
@@ -169,8 +169,8 @@ You may want to define your own survey if you wish to modify some parameters or
       "y": 9674.05,
   }
 
-  Rubin = btk.survey.Survey(
-      "Rubin",
+  LSST = btk.survey.Survey(
+      "LSST",
       pixel_scale=0.2,
       effective_area=32.4,
       mirror_diameter=8.36,
```

Comment:
Is this not outdated ?

Reply:
yup we need to update doc/notebooks still in #270